sar/README.md

```diff
@@ -337,11 +339,5 @@ For those details please refer to the ["Additional properties" column in the tab
 - 1.6.1 / 1.7.1: `SourceDataRepository` and `RepositoryURL` are covered by STAC link structures. 
   All CARD4L compliant STAC Catalog are **required** to make intensive use of STAC link relation types such as
   `root`, `parent`, `child`, `item` and `collection`.
-- 1.6.9: `PeakSideLobeRatio` and `IntegratedSideLobeRatio` are *recommended* to be included in the resource linked to with the relation type
-  [`performance-indicators`](#stac-item-links).
-- 1.7.9: The CARD4L specification is not clear whether a string representation of a CRS is required
```

Comment:
Has this been clarified by CEOS?

Reply:
In my view it is pretty clear that `PeakSideLobeRatio` and `IntegratedSideLobeRatio` are now target requirements. At least in the XLSX v5.5.5. The PDF v5.5 is a bit less clear: "Provide additional relevant performance indicators (**e.g.,** ENL, PSLR, ISLR **etc.**)".

Reply:
Yes, that is clear, but I was referring to 1.7.9 (CRS), not 1.6.9 (performance-indicators).

Reply:
Ah okay, I see what you mean. I'll re-add the note.

Reply:
I'll discuss 1.7.9 and 4.4 in the next meeting with CEOS (hopefully still in January) and then we can hopefully remove both of them in another PR.